multipers/multiparameter_module_approximation/approximation.h

```diff
@@ -1413,7 +1406,7 @@ inline std::vector<value_type> Module<value_type>::compute_pixels_of_degree(
   } // Timer death
   if (verbose)
     std::cout << "Module " << start->get_dimension()
-              << " has weight : " << moduleWeight << "\n";
+              << " has weight : " << moduleWeight << std::endl;
```

Comment:
?

Reply:
This adds a flush, which is not necessary here / should be handled by python

Reply:
If you want. But most of the times it means that it will just never be printed on a notebook...